pkgs/desktops/plasma-5/kwallet-pam.nix

```diff
@@ -4,7 +4,8 @@ mkDerivation {
   name = "kwallet-pam";
   nativeBuildInputs = [ extra-cmake-modules ];
   buildInputs = [ pam socat libgcrypt qtbase kwallet ];
-  postPatch = ''
-    sed -i pam_kwallet_init -e "s|socat|${lib.getBin socat}/bin/socat|"
+  qtWrapperArgs = [ "--prefix PATH : ${lib.getBin socat}/bin" ];
```

Comment:
I would prefer to leave the patch in place for the absolute path of `socat`. I don't like prefixing `PATH` when we know exactly what program we are looking for; there's not telling in the future what other programs might end up at that `PATH` entry.

Reply:
Seems reasonable, although the other side of this is we have a sed expression that may or may not always work.

Reply:
That's true. We should change that to `substituteInPlace` so the build will fail if it stops working. (Not necessary for this PR.)